test/extended/util/cluster/cluster.go

```diff
@@ -174,12 +196,29 @@ func SetTestConfig(platformStatus *configv1.PlatformStatus, masters, nonMasters
 func (c *ClusterConfiguration) GetMatchFn() func(string) bool {
 	var skips []string
 	skips = append(skips, fmt.Sprintf("[Skipped:%s]", c.ProviderName))
+
 	if c.NetworkPlugin != "" {
 		skips = append(skips, fmt.Sprintf("[Skipped:Network/%s]", c.NetworkPlugin))
 		if c.NetworkPluginMode != "" {
 			skips = append(skips, fmt.Sprintf("[Skipped:Network/%s/%s]", c.NetworkPlugin, c.NetworkPluginMode))
 		}
 	}
+
+	if !c.HasIPv4 {
+		skips = append(skips, "[Feature:Networking-IPv4]")
+	}
+	if !c.HasIPv6 {
+		skips = append(skips, "[Feature:Networking-IPv6]")
```

Comment:
just FYI these tags are not longer upstream, tests must work on both IP families, there is only one test remaining that I couldn't migrate

```
grep -r Networking  test | grep Fea
test/e2e/network/networking_perf.go:var _ = SIGDescribe("Networking IPerf2 [Feature:Networking-Performance]", func() {
test/e2e/network/networking.go: ginkgo.It("should provide Internet connection for containers [Feature:Networkin-IPv4]", func() {
test/e2e/network/networking.go: ginkgo.It("should provide Internet connection for containers [Feature:Networkin-IPv6][Experimental][LinuxOnly]", func() {
```

Reply:
heh, I just realise the tag is wrong Networkin/Networking